src/model.jl

```diff
@@ -6,9 +6,12 @@ mutable struct Model
     solar_forcing::Array{Float64,3}     # Time-dependent incoming solar radiation: depends on the orbital parameters and the albedo
     radiative_cooling_co2::Float64      # Constant outgoing long-wave radiation: depends on the CO2 concentration 
     radiative_cooling_feedback::Float64 # Outgoing long-wave radiation (feedback effects): models the water vapor cyces, lapse rate and cloud cover
-end
+    geography::Array{Int8,2}            
+    co_albedo::Array{Float64,2}
+    compute_albedo::Bool  
```

Comment:
Please remove `geography`, as it is not used throughout the simulation (we won't even even need it when the model is updated at every time step). You can leave `co_albedo` and `compute_albedo`, since I know that you will use them in your next PR, but please add a short explanatory comment (see, for example, the variables defined above).